docs/apis/calcite.md

```diff
@@ -0,0 +1,529 @@
+Apache Apex,unified stream processing engine enables application developers to process data at very high throughput with low latency. Though type of data demands different type of
+processing requirements but SQL remains way to process input data .To ensure that existing ETL developers,developers who are well versed with Database applications adopt stream processing appliction development with ease, integration of SQL  with Apex was needed. Being popular and Apache project, Calcite was chosen for an integration and to facilitate use of SQL with Apex.
+
+## APEX-Calcite Integrtion
+ Apache Calcite is highly customizable engine for parsing and planning queries on relational data from various data sources . Apache Calcite provides functionality of storage independent query optimization and ways to integrate them in other frameworks which would like to take advantage and expose SQL capability to their users.
+
+Particulary in Apex-Calcite integration, Calcite processes query and then creates Relational Algebra to create processing pipelines. These relational algebra processing pipelines
```

Comment:
These relational algebra processing pipelines are "converted to DAG" with set of operators to perform business logic on stream data